packages/loader/container-loader/src/container.ts

```diff
@@ -2415,83 +2415,86 @@ export class Container
 	): Promise<void> {
 		assert(this._runtime?.disposed !== false, 0x0dd /* "Existing runtime not disposed" */);
 
-		// The relative loader will proxy requests to '/' to the loader itself assuming no non-cache flags
-		// are set. Global requests will still go directly to the loader
-		const maybeLoader: FluidObject<ILoader> = this.scope;
-		const loader = new RelativeLoader(this, maybeLoader.ILoader);
-
-		const loadCodeResult = await PerformanceEvent.timedExecAsync(
-			this.subLogger,
-			{ eventName: "CodeLoad" },
-			async () => this.codeLoader.load(codeDetails),
-		);
+		try {
```

Comment:
This change is just adding a try .. catch block so that failures during container runtime initialization are caught and disposes the container.